lib/dangermattic/plugins/view_changes_need_screenshots.rb

```diff
@@ -17,6 +17,12 @@ class ViewChangesNeedScreenshots < Plugin
     VIEW_EXTENSIONS_IOS = /(View|Button)\.(swift|m)$|\.xib$|\.storyboard$/
     VIEW_EXTENSIONS_ANDROID = /(?i)(View|Button)\.(java|kt|xml)$/
 
+    IMAGE_IN_PR_BODY_PATTERNS = [
+      %r{https?://\S*\.(gif|jpg|jpeg|png|svg)},
+      /!\[(.*?)\]\((.*?)\)/,
+      /<img\s+[^\>]*src\s*=\s*[^\>]*>/
```

Comment:
I don't think it's necessary to escape the `>` inside the character class sub pattern, as it's not a special character interpreted other than a literal by Regexp… is it?

```suggestion
      /<img\s+[^>]*src\s*=\s*[^>]*>/
```